lib/policies/load-balancing.js

```diff
@@ -132,17 +132,22 @@ DCAwareRoundRobinPolicy.prototype.init = function (client, hosts, callback) {
   if (!this.localDc) {
     //get the first alive local, it should be local on top
     var hostsArray = hosts.values();
+    var dcHost;
     for (var i = 0; i < hostsArray.length; i++) {
       var h = hostsArray[i];
       if (h.datacenter) {
         this.localDc = h.datacenter;
+        dcHost = h;
         break;
       }
     }
     //this should never happen but it does not hurt
     if (!this.localDc) {
       return callback(new errors.DriverInternalError('Local datacenter could not be determined'));
     }
+    client.log('warning', 'No local DC was provided with DCAwareRoundRobinPolicy' + 
```

Comment:
Message may not be perfect, feedback welcome :)

Reply:
I think its clear enough.
I would use the word "data center" instead of the first occurrence of DC and replace the second occurrence of "provided" with "specified".

`'No local Data Center was provided with DCAwareRoundRobinPolicy'`
`'.  Future releases will require local DC to be specified.'`

Reply:
👍 will make those changes quickly.